pages/admin/criminal-check.tsx

```diff
@@ -0,0 +1,177 @@
+                    display="flex"
+                    marginTop="40px"
+                    paddingRight="20px"
+                    spacing="100px"
+                >
+                    <VStack
+                        width="400px"
+                        spacing="32px"
+                        marginLeft="40px"
+                        alignSelf="flex-start"
+                    >
+                        <HStack color={colourTheme.colors.Gray} width="full">
+                            <Icon as={MdPerson} w={8} h={8} />
+                            <VStack
+                                alignItems="flex-start"
+                                spacing="-3px"
+                                w="200px"
+                            >
+                                <Link>
+                                    {"Participant Information" +
+                                        " (" +
+                                        (props.volunteerName
+                                            ? props.volunteerName
+                                            : testData.volunteerName) +
+                                        ")"}
+                                </Link>
+                            </VStack>
+                        </HStack>
+                        <HStack
+                            fontWeight={700}
+                            color={colourTheme.colors.Blue}
+                            width="full"
+                        >
+                            <Icon as={MdDescription} w={8} h={8} />
+                            <Link>Criminal Record Check</Link>
+                        </HStack>
+                    </VStack>
+                    {criminalRecordLink !== undefined ? (
+                        <FileDownloadCard
+                            filePath={FILE_PATH}
+                            docLink={criminalRecordLink}
+                            docName={volunteerData.criminalRecordCheckLink}
+                            docApproved={volunteerData.criminalCheckApproved}
+                            participantEmail={
+                                props.volunteerEmail
+                                    ? props.volunteerEmail
+                                    : testData.volunteerEmail
+                            }
+                        />
+                    ) : (
+                        <Box
+                            width="full"
+                            height="200px"
+                            backgroundColor={colourTheme.colors.LightGray}
+                            display="flex"
+                            alignItems="center"
+                            justifyContent="center"
+                            textAlign="center"
+                            color={colourTheme.colors.Gray}
+                        >
+                            The participant has not uploaded a criminal record
+                            check at this time.
+                        </Box>
```

Comment:
We actually have a empty state component!
```suggestion
                        <EmptyState height="200px">
                            The participant has not uploaded a criminal record
                            check at this time.
                        </EmptyState>
```

To make it accept props, can you change the `/component/EmptyState.tsx` file to the following:
```typescript
import React from "react";
import { Box, Center, Flex, FlexProps } from "@chakra-ui/react";
import colourTheme from "@styles/colours";

type EmptyStateProps = FlexProps & {
    styleProps?: Record<string, unknown>;
};

export const EmptyState: React.FC<EmptyStateProps> = ({
    children,
    ...props
}): JSX.Element => {
    return (
        <Flex
            bg={colourTheme.colors.LightGray}
            w="100%"
            py={"40px"}
            px={"64px"}
            mt={5}
            display="flex"
            alignItems="center"
            justifyContent="center"
            textAlign="center"
            color={colourTheme.colors.Gray}
            {...props}
        >
            <Box>
                <Center whiteSpace="pre-line" textAlign="center">
                    {children}
                </Center>
            </Box>
        </Flex>
    );
};

```